README.md

```diff
@@ -1 +1,40 @@
-# Create-mobile-version
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# SET UP MOBILE VERSION
+
+> Description of the project
+The project was done to test the knowledge of linters, HTML, CSS and use of figma code
+## Built With
+
+- HTML
+- CSS
+
+
+
```

Comment:
- [ ] You should add the `Getting Started` section as in the README [template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse. You can add things like: `how to clone the repository`, `how to run it locally`, `the prerequisites to run it locally`, etc